modules/openapi-generator/src/main/resources/cpp-rest-sdk-client/api-header.mustache

```diff
@@ -15,6 +15,10 @@
 {{/imports}}
 {{^hasModelImport}}#include "../ModelBase.h"{{/hasModelImport}}
 
+#ifdef _WIN32 | _WIN64
```

Comment:
Small nit...
Would it make sense to add this in ApiClient header instead?

Reply:
Good call. I just updated my branch accordingly.